includes/MirahezeMagicHooks.php

```diff
@@ -254,4 +254,23 @@ public static function onSkinAddFooterLinks( Skin $skin, string $key, array &$fo
 			$footerItems['donate'] = $skin->footerLink( 'miraheze-donate', 'miraheze-donatepage' );
 		}
 	}
+
+	public static function onUserGetRightsRemove( User $user, array &$aRights ) {
+		$config = MediaWikiServices::getInstance()->getConfigFactory()->makeConfig( 'mirahezemagic' );
+		// Remove read from stewards on staff wiki.
+		if ( $config->get( 'DBname' ) === 'staffwiki' && $user->isLoggedIn() &&
+		    class_exists( 'CentralAuthUser' ) ) {
```

Comment:
Why not ExtensionRegistry->isLoaded ?

Reply:
Decided to remove the check as we check for the wiki we want this done on.